### PR TITLE
TNT-43949 Update ECID-service to fix NFE in parsing expiry values from AMCV cookie

### PR DESCRIPTION
**Pull request by XDex**
## Description

Update ECID-service to v1.0.3 in order to fix NFE in parsing expiry values from AMCV cookie
Note: @shandilya3 - this PR also reverts the change from #57 adding support for ClientHints in the Context Delivery request DTO, as we can't release it until support for ClientHints has been released on the Target edge. We will need to re-apply this change in the upcoming v2.2.3 SDK release adding support for ClientHints.

## Related Issue

https://jira.corp.adobe.com/browse/TNT-43949

## Motivation and Context

Update ECID-service to v1.0.3 in order to fix NFE in parsing expiry values from AMCV cookie

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,13 +1,16 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [2.2.2] - 2022-03-30
+### Fixed
+- Updated ECID-service dependency to fix parsing expiry values from AMCV cookie
+- Add ClientHints to Delivery API model in preparation for ClientHints support
```
Comment (dcottingham) [inline CHANGELOG.md:7]: Are the client hints models included in 2.2.1?

**Review by dcottingham — APPROVED** [review]
Just one comment, but looks good otherwise.

**Review by shandilya3 — APPROVED** [review]
+1 , Just change change-log comment

File: CHANGELOG.md (hunk 1 above)
Comment (XDex) [reply CHANGELOG.md:7]: @dcottingham I've only removed the reference to ClientHints in `Context` DTO, but the actual `ClientHints` DTO added in #57 remains, so I think we should still mention it in the changelog..

**Pull request merged**